lib/queue_classic_plus/update_metrics.rb

```diff
@@ -18,7 +18,7 @@ def self.metrics
       {
         jobs_queued: jobs_queued,
         jobs_scheduled: jobs_scheduled,
-        max_created_at: max_age("created_at"),
+        max_created_at: max_age("created_at", "created_at = scheduled_at"),
```

Comment:
If we do this, when the scheduled time is passed but the job not executed, it's going to report big numbers. That's what leads to big spikes in our Librato metrics.